src/OrbitGl/App.h

```diff
@@ -423,6 +423,9 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
                                                           const std::string& build_id);
   [[nodiscard]] orbit_base::Future<ErrorMessageOr<void>> LoadSymbols(
       const std::filesystem::path& symbols_path, const std::string& module_file_path);
+
+  [[nodiscard]] orbit_base::Future<ErrorMessageOr<std::filesystem::path>> LoadModuleOnRemote(
```

Comment:
The other `LoadModuleOnRemote` will be removed but I still find it weird that the two are not next to each other in this intermediate state.

Reply:
Yes, good point. Fixed. Thanks!